frontend/src/features/books/BookList.jsx

```diff
@@ -52,7 +52,8 @@ return (
               <td style={{ padding: '10px' }}>{book.title}</td>
               <td style={{ padding: '10px' }}>{book.author}</td>
```

Comment:
@nshelokar-vf, I see the style `{ padding: '10px' }` is being used in multiple places.
Can we avoid inline CSS + reuse the CSS class?